Formula/flamegraph.rb

```diff
@@ -0,0 +1,66 @@
+class Flamegraph < Formula
+  desc "Stack trace visualizer"
+  homepage "https://github.com/brendangregg/FlameGraph"
+  url "https://github.com/brendangregg/FlameGraph/archive/v1.0.tar.gz"
+  sha256 "c5ba824228a4f7781336477015cb3b2d8178ffd86bccd5f51864ed52a5ad6675"
+  head "https://github.com/brendangregg/FlameGraph.git"
+
+  depends_on "perl"
```

Comment:
Can we use default perl from macOS? 

If so, then
```suggestion
  uses_from_macos "perl"
```